python/ray/worker.py

```diff
@@ -907,6 +909,7 @@ def _process_task(self, task):
 
         # Get task arguments from the object store.
         try:
+            self.reraise_actor_init_error()
```

Comment:
Hmm we probably will need to special case the terminate method otherwise you can never destroy failed actors.

Reply:
Fixed.